plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/machine/Machine.java

```diff
@@ -75,7 +75,7 @@ public String getRecipeUrl() {
         MachineSourceDto machineSource = descriptor.getConfig().getSource();
 
         String machineSourceType = machineSource.getType();
-
+// todo
```

Comment:
Do we have issue for that? Add number of it or just add more information What will be done later